po/fr.po

```diff
@@ -1,21 +1,21 @@
-# Nicolas Martyanoff <khaelin@gmail.com>, 2007–2008
-# Sabrina Dubroca <sd@queasysnail.net>, 2013
-# rugie <fliehen@posteo.net>, 2017
 #
+# Nicolas Martyanoff <khaelin@gmail.com>, 2007–2008.
+# Sabrina Dubroca <sd@queasysnail.net>, 2013.
+# rugie <fliehen@posteo.net>, 2017, 2022.
+# Tonus <tonus1@gmail.com>, 2022.
 msgid ""
 msgstr ""
 "Project-Id-Version: newsboat 2.6\n"
 "Report-Msgid-Bugs-To: https://github.com/newsboat/newsboat/issues\n"
 "POT-Creation-Date: 2021-12-08 18:49+0300\n"
-"PO-Revision-Date: 2020-12-10 21:26+0100\n"
-"Last-Translator: Tanguy Kerdoncuff <t.kerdonc@gmail.com>\n"
-"Language-Team: Nicolas Martyanoff <khaelin@gmail.com, Tanguy Kerdoncuff <t."
-"kerdonc@gmail.com>\n"
+"PO-Revision-Date: 2022-03-03 14:28+0000\n"
+"Last-Translator: antoine <tonus1@gmail.com>\n"
+"Language-Team: French <kde-i18n-doc@kde.org>\n"
```

Comment:
We don't have language teams in Newsboat, so we enumerate all the translators in this field. Please roll back this change, and add yourself.